apprise/plugins/msteams.py

```diff
@@ -176,6 +179,15 @@ class NotifyMSTeams(NotifyBase):
             'required': True,
             'regex': (r'^[a-z0-9-]+$', 'i'),
         },
+        # Token required as part of the API request
+        #  /........./........./........./DDDDDDDDDDDDDDDDD
+        'token_d': {
+            'name': _('Token D'),
+            'type': 'string',
+            'private': True,
+            'required': False,
+            'regex': (r'^V2[a-zA-Z0-9-_]+$', 'i'),
```

Comment:
This pattern could be relaxed - I have just observed that any new webhook URLs have a token here that starts with `V2`